### PR TITLE
add missing secrets and values

### PR DESCRIPTION
**Pull request by ntarocco**

### CODE REVIEW

**Comment by ntarocco** [comment]
@slint @ppanero @jrcastro2 @avivace changes are requires in your instances.
Take into account and probably apply changes:
- the new base image `FROM registry.cern.ch/inveniosoftware/almalinux:1`
- it should be possible to use the command `invenio webpack buildall` instead of the [3 separated commands](https://github.com/inveniosoftware/cookiecutter-invenio-rdm/pull/249/files#diff-6ecfe44e1f8b1bf38722e8c15b4846a9d8b2852e7f6dae4852848b3eff5c964fL33). We will have to change `invenio-cli` too. Still to be tested well.
- from this PR:
    - missing secrets `SECURITY_LOGIN_SALT` and `CSRF_SECRET_SALT` (including injection in deployment configs)
    - `haproxy.image: "registry.cern.ch/docker.io/library/haproxy:2.2"`
    - `nginx.image: "registry.cern.ch/docker.io/library/nginx:1.23"`
    - `redis.image: "registry.cern.ch/docker.io/library/redis:7"`
    - `rabbitmq.image: "registry.cern.ch/docker.io/library/rabbitmq:3-management"`
    - small changes to `HaProxy` config, but nothing critical (split max conn between backend and static)
    - make mail settings configurable
    - improve readiness/liveness probes based on our experience
 For our instance, I have created a private repo to store our Helm charts **values** and make the configuration reproducible. Ask me for access if interested.

File: invenio/values.yaml [hunk 1]
```diff
@@ -28,6 +26,11 @@ invenio:
       level: "WARNING"
       celery: true
       environment: "qa"
+  mail:
+    suppress: true
```
Comment (avivace) [inline invenio/values.yaml:30]: is this different from `enabled`? (which we may seem to use to decide if inject or not mail configurations in `invenio/templates/configurations/invenio.yaml`)

**Review by avivace — COMMENTED** [review]
Thanks a lot @ntarocco ! Lots of needed changes. I will try it and report back but in the meantime I just left two minor comments.
Another question: are the currently set probes for the invenio pod working out of the box for you? They seemed to always fail to me

File: invenio/templates/deployments/rabbitmq.yaml [hunk 2]
```diff
@@ -45,9 +45,23 @@ spec:
...
             exec:
               command:
-                - sh
-                - -c
-                - "rabbitmq-diagnostics -q check_running && rabbitmq-diagnostics -q check_local_alarms"
-            initialDelaySeconds: 15
+                - /bin/bash
+                - '-c'
+                - rabbitmq-diagnostics -q check_running && rabbitmq-diagnostics -q check_local_alarms
+            initialDelaySeconds: 25
```
Comment (kpsherva) [inline invenio/templates/deployments/rabbitmq.yaml:51]: 👍

**Review by kpsherva — APPROVED** [review]

**Review by ppanero — APPROVED** [review]

File: invenio/values.yaml (hunk 1 above)
Comment (ntarocco) [reply invenio/values.yaml:30]: Thank you for spotting this! The `enabled` was wrong. I have removed it! :)

**Pull request merged**